pkg/types/azure/machinepool.go

```diff
@@ -36,6 +36,12 @@ type OSDisk struct {
 	DiskType string `json:"diskType"`
 }
 
+// DefaultDiskType holds the default Azure disk type used by the VMs.
+const DefaultDiskType string = "Premium_LRS"
+
+//DefaultInstanceType holds the default Azure instance type used by VMs.
+const DefaultInstanceType string = "Standard_D4s_v3"
```

Comment:
I steered you wrong on this constant. You should be calling these functions instead.
https://github.com/openshift/installer/blob/eb47ed1dea8b4ac393e58d1b6bd19d7edb113908/pkg/types/azure/defaults/machines.go#L19
https://github.com/openshift/installer/blob/eb47ed1dea8b4ac393e58d1b6bd19d7edb113908/pkg/types/azure/defaults/machines.go#L27